From 1f18ad8724a2be06821ac634916d36c3caa86e8c Mon Sep 17 00:00:00 2001
From: androiddevelopserver <liyong.kang@hytera.com>
Date: Mon, 5 Sep 2022 17:01:13 +0800
Subject: [PATCH] src m105 8be39b46038d6ac675d9fd55735ceb1e5f1ea63e

---
 rtc_base/BUILD.gn | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/rtc_base/BUILD.gn b/rtc_base/BUILD.gn
index e74b85fcbf..0dbdf0d67d 100644
--- a/rtc_base/BUILD.gn
+++ b/rtc_base/BUILD.gn
@@ -18,7 +18,19 @@ if (!rtc_build_ssl) {
   config("external_ssl_library") {
     assert(rtc_ssl_root != "",
            "You must specify rtc_ssl_root when rtc_build_ssl==0.")
-    include_dirs = [ rtc_ssl_root ]
+    include_dirs = [ "$rtc_ssl_root/include" ]
+    if( is_win ) {
+      libs = [
+        "$rtc_ssl_root/lib/libcrypto.lib",
+        "$rtc_ssl_root/lib/libssl.lib"
+      ]
+    }
+    if( is_mac || is_linux || is_ios || is_android ) {
+      libs = [
+        "$rtc_ssl_root/lib/libssl.a",
+        "$rtc_ssl_root/lib/libcrypto.a",
+      ]
+    }
   }
 }
 
-- 
2.25.1

